src/flux/task-plan.coffee

```diff
@@ -118,7 +110,8 @@ TaskPlanConfig =
     {tasking_plans} = @_getPlan(id)
     # do all the tasking_plans have the same date?
     dates = _.compact _.uniq _.map(tasking_plans, (plan) ->
-      plan[attr]?.getTime()
+      date = new Date(plan[attr])
+      date?.getTime()
     )
```

Comment:
Date will always be defined here, so the `?` has no effect.  If a Date object is created with an invalid value it still creates the `Date` object, but it's value is set to `Invalid Date`.  One method to detect if the date is invalid is to use `isNaN`  To use that method you could write line 114 as: `if isNaN(d.getTime()) then 0 else d.getTime()`